controllers/account.go

```diff
@@ -291,7 +291,7 @@ func (c *ApiController) Logout() {
 			}
 
 			c.ClearUserSession()
-			object.DeleteSessionId(user, c.Ctx.Input.CruSession.SessionID())
+			object.DeleteSessionId(util.GetSessionId("built-in", "app-built-in", user), c.Ctx.Input.CruSession.SessionID())
```

Comment:
The api `logout` is also used for third-party application logout according openid protocol. Here should use `application.name` `application.organization`. 

If it's not the main content of the pr. I'll fixed it after the pr merged.